g3doc/quick_reference.md

```diff
@@ -1385,7 +1385,7 @@ instead because they are more general:
     must be in the range `[0, 2 * Lanes(d))` but need not be unique. The index
     type `TI` must be an integer of the same size as `TFromD<D>`.
 
-*   `V`: `{u,i,f}{16,32,64}` \
+*   `V`: `{u,i}{8,16}; {u,i,f}{32,64}` \
```

Comment:
Now that all types are supported, which is great news, we can remove this line because there are no more constraints on V.